RuleDocumentation/PossibleIncorrectUsageOfComparisonOperator.md

```diff
@@ -0,0 +1,7 @@
+# PossibleIncorrectUsageOfComparisonOperator
+
+**Severity Level: Information**
```

Comment:
I was wondering because we can determine pretty accurately know that the usage of the comparison operator inside if statements is unintentional, if we should up the level to warning because some people ignore/disable warnings of informational level?